spec.md

```diff
@@ -425,6 +426,39 @@ encapsulated within the `data` attribute.
 - Constraints:
   - OPTIONAL
 
+# Size Limits
+
+In many scenarios, CloudEvents will be forwarded through one or more
+generic intermediaries, each of which might impose limits on the size of
+forwarded events. CloudEvents might also be routed to consumers, like
+embedded devices, that are storage or memory-constrained and therefore
+would struggle with large singular events.
+
+The "size of an event" is it's wire-size, and includes every bit that is
+transmitted on the wire for the event: transport frame-metadata, event
+metadata, and event data, based on the chosen event format and the chosen
+protocol binding.
+
+If an application configuration requires for events to be routed across
+different transports or for events to be re-encoded, the least efficient
+transport and encoding used by the application SHOULD be considered for
+compliance with these size constraints:
+
+- Intermediaries MUST forward events of a size of at least 64 KByte.
```

Comment:
Although I believe I understand your intent, this technically only creates a requirement for events of "at least 64 KByte" or in other words, events with a size that is 64KB or more.  As such, as written it doesn't discuss events of the smaller size but worse, it specifies that all events 64KB or **larger** MUST be forwarded by intermediaries.

Reply:
@erikerikson can you suggest some alternate wording?

Reply:
My original recommendation was offered in a previous [comment] (https://github.com/cloudevents/spec/pull/405#discussion_r268255924) but new framing has been developed which is good.  I'll attempt to make a suggestion in the next few days.

Reply:
`.... of a size of 64 KBytes or less.`